build_projects/dotnet-cli-build/PublishTargets.cs

```diff
@@ -19,160 +19,115 @@ public static class PublishTargets
 
         private static DebRepoPublisher DebRepoPublisherTool { get; set; }
 
+        private static VersionRepoUpdater VersionUpdater { get; set; }
+
         private static string Channel { get; set; }
 
+        private static string Pointer { get; set; }
+
         private static string CliVersion { get; set; }
 
         private static string CliNuGetVersion { get; set; }
 
-        private static string SharedFrameworkNugetVersion { get; set; }
-
         [Target]
         public static BuildTargetResult InitPublish(BuildTargetContext c)
         {
             AzurePublisherTool = new AzurePublisher();
             DebRepoPublisherTool = new DebRepoPublisher(Dirs.Packages);
+            VersionUpdater = new VersionRepoUpdater(EnvVars.EnsureVariable("GITHUB_PASSWORD"));
 
             CliVersion = c.BuildContext.Get<BuildVersion>("BuildVersion").SimpleVersion;
             CliNuGetVersion = c.BuildContext.Get<BuildVersion>("BuildVersion").NuGetVersion;
-            SharedFrameworkNugetVersion = DependencyVersions.SharedFrameworkVersion;
             Channel = c.BuildContext.Get<string>("Channel");
+            Pointer = "Latest";
 
             return c.Success();
         }
 
         [Target(nameof(PrepareTargets.Init),
         nameof(PublishTargets.InitPublish),
         nameof(PublishTargets.PublishArtifacts),
-        nameof(PublishTargets.TriggerDockerHubBuilds),
-        nameof(PublishTargets.FinalizeBuild))]
+        nameof(PublishTargets.FinalizeBuild),
+        nameof(PublishTargets.TriggerDockerHubBuilds))]
```

Comment:
I believe @MichaelSimons was going to work on moving this into FinalizeBuild.

Any reason why these are being switched around?